### PR TITLE
no more warnings in the tests

### PR DESCRIPTION
**Pull request by zakird**
Cleans up warnings that were coming up in the unit tests

### CODE REVIEW

File: zschema/tests.py [hunk 1]
```diff
@@ -17,10 +17,14 @@ class LeafUnitTests(unittest.TestCase):
...
     def test_valid(self):
         for leaf in VALID_LEAVES:
+            if leaf.DEPRECATED:
+                continue
             leaf().validate(leaf.__name__, leaf.VALID)
 
     def test_invalid(self):
         for leaf in VALID_LEAVES:
+            if leaf.DEPRECATED:
+                continue
```
Comment (cdzombak) [inline zschema/tests.py]: Why are we just excluding deprecated leaves from tests? It seems to me that either they should be removed, or they should continue to be tested with the rest of the codebase.

File: zschema/tests.py [hunk 2]
```diff
@@ -149,7 +153,7 @@ def setUp(self):
         })
         self.host = Record({
                 "ipstr":IPv4Address(required=True),
-                "ip":Long(),
+                "ip":Unsigned32BitInteger(),
```
Comment (cdzombak) [inline zschema/tests.py]: I assume this change (and the similar change `example.py`) is because `Long` is deprecated?

**Comment by zakird** [comment]
Yeah that's totally fair, and bothered me slightly too. We're probably at
the point where we can just remove them. How about I close this, make sure
our schemas don't have any, then remove?

**Comment by cdzombak** [comment]
> How about I close this, make sure
> our schemas don't have any, then remove?

👍

**Pull request closed without merging**